SilasKenneth/fast-food-fast-ui
# [Finishes #160135903] Admin user should be able to view items

#### What this pull request does
Creates the page on the admin dashboard where all items are listed
##### Description of task to be completed
 Create a page where all products are listed
    -  Add links items orders account and log out
    - Create the view for the items
    - Create a button to add new items
##### Relevant pivotal tracker story
[#160135903](https://www.pivotaltracker.com/n/projects/2194200/stories/160135903)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-list-items-160135903```
  - Open the items.html file in your browser and that should work

#### Screenshot
![List all items admin](https://user-images.githubusercontent.com/11555354/44910238-1c03a500-ad2b-11e8-8e4a-de3f525914e9.png)
